firefox_puppeteer/tests/test_menubar.py

```diff
@@ -20,7 +17,8 @@ def test_click_item_in_menubar(self):
 
         def opener(_):
             # Hard-coded labels will not work in localized builds
```

Comment:
nit: lets remove that line. It was only a temporary placeholder to visualize that this won't work.